docker-compose.yaml

```
@@ -0,0 +1,10 @@
+version: "3.8"
+
+services:
+
+  movieweb:
+    build:
+      context: .
+    ports:
+      - "80:80"
+    restart: always
```

```suggestion
    restart: unless-stopped
```
we do not want that the container starts after we stopped it

---

Good spot, thanks @DenuxPlays